srml/democracy/src/lib.rs

```diff
@@ -933,9 +931,9 @@ impl<T: Trait> Module<T> {
 			if info.delay.is_zero() {
 				Self::enact_proposal(info.proposal, index);
 			} else {
-				<DispatchQueue<T>>::mutate(
+				<DispatchQueue<T>>::safe_append(
```

Comment:
this looks like `extend` (or `extend_from_slice`). also, is there an `unsafe_append`?

Reply:
the normal `append()` is the unsafe one. 

Though I wouldn't call it actually __unsafe__; if the data is corrupt and non-decodable, it will return an error -- I don't see this as an __unsafe behaviopr__. The other one (currently named `safe_append`) will simply `put()` if the data is corrupt. 

I will apply your recommendation of `append()` and `append_or_put()` since it is more clear.